src/Modals/__tests__/Modal.test.tsx

```diff
@@ -50,6 +61,51 @@ describe('Modal', function () {
         strictEqual(modal.getAttribute('aria-labelledBy'), 'facultyEditButton');
       });
     });
+    context('when forwardRef prop is present', function () {
+      beforeEach(function () {
+        const RefExample = () => {
+          const ref = useRef(null);
+          const [modalVisible, setModalVisible] = useState(false);
+          const onButtonClick = () => {
+            setModalVisible(true);
+            setTimeout(() => ref.current.focus(), 0);
```

Comment:
setTimeout(() => ref.current.focus());

Reply:
Just fixed this through 4597089.

Reply:
Thanks for catching this. Fixed this through 95e03a8!